### ISSUE TITLE
Cross link input

### ISSUE THREAD

**Issue by roivant-matts** [Rappsilber-Laboratory/AlphaLink2#3]
Hello, Thanks for this excellent project. For technical reproducibility, are you able to share the RpoA-RpoC cross-links csv or pkl dictionary? I wasn't able to find in the paper/supplements etc

**Comment by roivant-matts** [comment]
Thanks and I gather the CSV format is 1-based, but the dictionary is 0 based? I guess expected, but I needed to add FDR values to above to get the prediction.

**Comment by lhatsk** [comment]
Yes, CSV format is 1-based, and dictionary 0-based. Sorry, about the missing FDR, my internal set-up is a little different and has a fixed FDR.

**Comment by roivant-matts** [comment]
Thanks - I am rerunning to be sure I didn't have a mixup, but I found using the v2 params with and without cross-links (e.g. an empty {} pkl.gz) I get the same structure. (both match the reference PDB closely - e.g. superimposing al2 chain B (rpoc) on chain D of the reference). Is my approach to use an empty dictionary as a baseline appropriate in your view?

**Comment by lhatsk** [comment]
Yes, that works as a baseline. We used an FDR of 0.05 but it doesn't matter here.

We noticed the same thing last week. Increasing the crop size during fine-tuning seems to already improve the RpoA-RpoC prediction sometimes. In our runs, 5/10 failed without crosslinks, whereas 10/10 succeeded with crosslinks so similar to the other experiments they allow us to focus sampling on the interesting regions. 

On the CASP data, it didn't seem to have a big effect (see extended data figure 3 in the v2 paper supplement).

**Comment by roivant-matts** [comment]
For the figure 3 data do you recall if the v2.2.4 or v2.3.0 weights are used for the alphafold predictions? I noticed in their release notes for v2.3.0 they also increased crop size to 640AA. Apologies for lag in coming back to this thread - some other testing brought it back to my mind.

**Comment by lhatsk** [comment]
Sorry for the late response!

For Figure 3 (the cullin4 data) we switched to v2.3.0 because the other networks were not able to produce meaningful predictions. v2.3.0 performs much better for larger complexes.
